src/main/java/mate/academy/dbconnection/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.dbconnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
The DB_URL is incomplete and missing the actual database URL. It should be a full JDBC URL of the form jdbc:mysql://<host>:<port>/<database_name>, for example: "jdbc:mysql://localhost:3306/my_database".